components/DocumentModels.vue

```diff
@@ -0,0 +1,150 @@
+<script setup lang="ts">
+import type { IAllDocumentModels, TrackModel } from "@bcc-code/bmm-sdk-fetch";
+import type { IDiscoverableGroup } from "~/composables/discover";
+import { breakpointsTailwind, useBreakpoints } from "@vueuse/core";
+
+const { t } = useI18n();
+
+const props = withDefaults(
+  defineProps<{
+    type?: "page" | "playlist" | "private-playlist";
```

Comment:
This property doesn't seem to be used anywhere. Is it needed?

Reply:
removed